README.md

```diff
@@ -2,56 +2,168 @@
+> If you are new to Nextflow, please refer to [this page](https://www.nextflow.io/docs/latest/install.html#installation) on how to set-up Nextflow. Make sure to test your setup before running the workflow on actual data.
 
-<!-- TODO nf-core: Describe the minimum required steps to execute the pipeline, e.g. how to prepare samplesheets.
-     Explain what rows and columns represent. For instance (please edit as appropriate):
+2. **Install Container Engine**:
+   - **Docker**: Follow [Docker installation guide](https://docs.docker.com/get-docker/)
+   - **Singularity**: Follow [Singularity installation guide](https://sylabs.io/guides/3.0/user-guide/installation.html)
+
+3. **Verify Installation**:
+   ```bash
+   nextflow info
+   docker --version  # or singularity --version
+   ```
+
+4. **Obtain PCGL Access Token**:
+   - Contact your PCGL administrator or data coordinator
+   - Ensure the token has appropriate permissions for your study
+
+## Usage
 
-First, prepare a samplesheet with your input data that looks as follows:
+For more detailed usage instructions, including additional examples and configuration options, see **[Complete Usage Guide](docs/usage.md)** which contains detailed command-line examples, instructions, and advanced configurations.
 
-`samplesheet.csv`:
+First, prepare your data and metadata files into a data directory structure, e.g:
 
-```csv
-sample,fastq_1,fastq_2
-CONTROL_REP1,AEG588A1_S1_L002_R1_001.fastq.gz,AEG588A1_S1_L002_R2_001.fastq.gz
+```
+input/
+├── metadata/
+│   ├── file_metadata.tsv          # Required: File information  
+│   ├── analysis_metadata.tsv      # Required: Analysis details
+│   ├── workflow_metadata.tsv      # Optional: Workflow information
+│   ├── read_group_metadata.tsv    # Optional: Read group details
+│   ├── experiment_metadata.tsv    # Optional: Experiment information
+│   ├── specimen_metadata.tsv      # Optional: Specimen details
+│   └── sample_metadata.tsv        # Optional: Sample information
+└── data/
+    └── [your data files - CRAM, BAM, etc.]
 ```
 
-Each row represents a fastq file (single-end) or a pair of fastq files (paired end).
 
--->
+### Basic Usage with Required Metadata
 
-Now, you can run the pipeline using:
+```bash
+nextflow run Pan-Canadian-Genome-Library/molecular-data-submission-workflow \
+    --study_id "YOUR_STUDY_ID" \
+    --token "YOUR_ACCESS_TOKEN" \
+    --path_to_files_directory "/path/to/data" \
+    --file_metadata "/path/to/file_metadata.tsv" \
+    --analysis_metadata "/path/to/analysis_metadata.tsv" \
+    --outdir results \
+    -profile docker,sd4h_prod
+```
 
-<!-- TODO nf-core: update the following command to include all required parameters for a minimal example -->
+### Advanced Usage with Additional Metadata
 
 ```bash
 nextflow run Pan-Canadian-Genome-Library/molecular-data-submission-workflow \
```

Comment:
```
The following step will attempt to submit files and their corresponding analysis and biospecimen metadata. New records will be registered. Existing records will be verified and flagged for discrepancies and otherwise cross referenced and skipped.
```